source/scripting_v3/GTA/Entities/Peds/Task/TaskInvoker.cs

```diff
@@ -752,6 +752,31 @@ private void PlayAnimationInternal(ClipDictionary clipDict, string animName, flo
 				rotation.X, rotation.Y, rotation.Z, blendInDeltaArg, blendOutDeltaArg, timeToPlay, (int)flags,
 				startPhase, (int)rotOrder, (int)ikFlags);
 		}
+
+		/// <summary>
+		/// Sets the speed of the anim task matching the given ClipDictAndAnimNamePair on the <see cref="Ped"/> to the given multiplier.
+		/// </summary>
+		/// <param name="clipDictAndAnimNamePair">The pair of a clip/animation dictionary name and an animation name.</param>
+		/// <param name="speedMultiplier">The amount to multiply the animation's normal playback speed by.</param>
+		public void SetAnimationSpeed(ClipDictAndAnimNamePair clipDictAndAnimNamePair, float speedMultiplier)
+		{
+			(ClipDictionary clipDict, string animName) = clipDictAndAnimNamePair;
+
+			Function.Call(Hash.SET_ENTITY_ANIM_SPEED, _ped.Handle, clipDict, animName, speedMultiplier);
+		}
+
+		/// <summary>
+		/// Sets the timestamp of the anim task matching the given ClipDictAndAnimNamePair on the <see cref="Ped"/> to the given value.
+		/// </summary>
+		/// <param name="clipDictAndAnimNamePair">The pair of a clip/animation dictionary name and an animation name.</param>
+		/// <param name="timestamp">The timestamp in the animation to set the specified animation to.</param>
+		public void SetAnimationTime(ClipDictAndAnimNamePair clipDictAndAnimNamePair, float timestamp)
+		{
+			(ClipDictionary clipDict, string animName) = clipDictAndAnimNamePair;
+
+			Function.Call(Hash.SET_ENTITY_ANIM_SPEED, _ped.Handle, clipDict, animName, timestamp);
```

Comment:
You mean `SET_ENTITY_ANIM_CURRENT_TIME`? We could rename the method to `SetAnimationCurrentTime` as there's `GetAnimationCurrentTime` in Entity class.

Reply:
Oops! Good catch, thank you for that.